files/build_templates/docker_image_ctl.j2

```diff
@@ -261,10 +287,15 @@ stop() {
 
 OP=$1
 DEV=$2 # namespace/device number to operate on
+NAMESPACE_PREFIX="asic"
 if [ "$DEV" ]; then
-    NET_NS="asic$DEV" #name of the network namespace
-else
+    NET_NS="$NAMESPACE_PREFIX$DEV" #name of the network namespace
+    SONIC_CFGGEN="sonic-cfggen -n $NET_NS"
```

Comment:
>sonic-cfggen -n $NET_NS" [](start = 18, length = 24)

Also. Why not merge the two branches into a unified.
What is the gap to use `sonic-cfggen -n ''` directly?

Reply:
I have been experimenting with adding -n '' directly instead of having the definition as above. There are couple of issues I see,

> When -n "namespace" option is provided ( even if it is -n ''), the SonicV2Connector uses UNIX socket thinking it is a connection to a DB in a different context. This requires the need for using sudo when invoking those scripts.
> If we add -n '' directly, It is getting used for the single ASIC platform ( uses the unix socket for connection ) which Ideally we don't want to have any change in behavior of existing platforms.

So I have kept the original way of invoking the sonic-db-cli and sonic-cfggen based on whether $DEV is '' or a valid instance.  I have done tests in single ASIC platform also and all the changes are done so that the single ASIC behavior is untouched.